app/gameengine.py

```diff
@@ -2745,12 +2833,15 @@ def is_condition_met(self, effect_player: PlayerState, source_card_id, condition
                 if len(effect_player.collab) == 0:
                     return False
                 return effect_player.collab[0]["game_card_id"] == source_card_id
+            case Condition.Condition_ThisCardIsPerforming:
+                return self.performance_performer_card["game_card_id"] == source_card_id
```

Comment:
in this case, there's probably no way you can do this while performance_performer_card is still None.
But it does start as None, so let's handle that anyway.

Reply:
Can you humor me and add this check? just an additional self.performance_performer_card and ...

Reply:
Oops. Sorry, I missed this one. I knew I was forgetting something.